### DISCUSSION TITLE
[QUESTION] Is it possible to use both dataDirs and torrentDir at the same time?

### DISCUSSION THREAD

**Discussion by pentago** [cross-seed/cross-seed#547] [Q&A] [answered]
I initially configured a data-based search and it works fine but I noticed that torrents from [autobrr cross-seed filter](https://autobrr.com/3rd-party-tools#cross-seed-filter) stopped coming in.

Can I use both `dataDirs` and `torrentDir` at the same time so cross-seed looks for torrents I already have on disk and at the same time use autobrr cross-seed filter to start seeding torrents that appear on IRC announcement channels which I already have in my client?

Or I can use only data-based search OR autobrr cross-seed filter?

I figure my question and use case is possibly weird, I just wanted to utilize my seeding potential to the max.

Thanks in advance!

**Comment by zakkarry** [comment, marked as answer]
This is mentioned in the [FAQ](https://www.cross-seed.org/docs/basics/faq-troubleshooting#my-data-based-search-is-searching-torrent-files) albeit not explicitly for your use-case I guess.

If you have a dataDirs set, and torrentDir is obviously mandatory - although it could be empty, it will search both when a general search (scheduled or `cross-seed search` command) is conducted (based on your include and exclude settings).

Torrent files will take precedence over data files, so if they share the same name the torrent will be used.

As far as autobrr goes, it will not search your dataDirs for files when sending to the announce endpoint ([docs reference](https://www.cross-seed.org/docs/reference/api#post-apiannounce)) - it will search your torrents.

There is no way to reverse data search with RSS or announce API.

**Reply by pentago** [reply]
Yeah, I'm aware that autobrr won't use data-based search and that is fine.

I just wondered, if I used both approaches, could I benefit from data-based search as well as from cross-seed filter set in autobrr?
I don't mind which search takes precedence.

**Comment by zakkarry** [comment]
Yes, I'd recommend using autobrr and setting something up from [this](https://www.cross-seed.org/docs/basics/daemon#set-up-automatic-searches-for-finished-downloads) part of the docs.

Using data matching for the complete searches and IRC announces through autobrr would be my preference.
